edx-olive-oc/edx-platform
# Update translated LMS djangojs.js files

This step was missed before merging https://github.com/edx-olive-oc/edx-platform/pull/9

**Testing instructions**

This issue is difficult to test in the devstack because the issue doesn't occur in development; it only occurs in production when the static asset pipeline is in use.

But you can verify the conditions by running these steps, and ensuring that no changes are made on this branch:
```bash
./manage.py lms --settings openstack compilemessages
./manage.py lms --settings openstack compilejsi18n
./manage.py cms --settings openstack compilejsi18n
```

**Reviewer**
